tools/tests/radial-menu/test.js

```diff
@@ -129,17 +131,18 @@ describe('Radial Menu Tests', () => {
 			setupRadialMenuTestPage().then(done).catch(err => console.error(err));
 		});
 
-		xit('Should have opened on keydown', function (done) {
+		it('Should have opened on keydown', function (done) {
 			const radialMenu = document.querySelector('gameface-radial-menu');
 
-			dispatchKeyboardEventKeyDown(16, radialMenu);
+			dispatchKeyboardEventKeyDown(16, window);
 
 			// The menu is opened 2 frames after the opening function is called.
-			// More time is apparently needed for it to open and 200 frames seems to not be flaky.
-			waitForStyles(() => {
-				assert(radialMenu.classList.contains('radial-menu-open') === true, 'Radial menu does not have class radial-menu-open.');
-				done();
-			}, 10);
+			requestAnimationFrame(() => {
+				requestAnimationFrame(() => {
```

Comment:
In feature we need to expose those waits in a separate module or use the method from the components library.

Reply:
Changed with `waitForStyles()` exposed from actions.js